tv_apps/smart-system/js/fxa_client.js

```diff
@@ -0,0 +1,140 @@
+/* -*- Mode: Java; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- /
```

Comment:
Did you changed the filename? I guess this fire corresponds to `apps/system/fx_accounts_client.js`?